fe/fe-core/src/main/java/com/starrocks/common/util/LogUtil.java

```diff
@@ -45,4 +49,11 @@ public static void logConnectionInfoToAuditLogAndQueryQueue(ConnectContext ctx,
         queryDetail.setErrorMessage(ctx.getState().getErrorMessage());
         QueryDetailQueue.addAndRemoveTimeoutQueryDetail(queryDetail);
     }
+
+    public static String getCurrentStackTrace() {
+        TimeoutException e = new TimeoutException("lock acquisition timeout");
```

Comment:
should not use TimeoutException, if this is a generic method to grape the current thread trace, 

use `Thread.currentThread().getStackTrace();` or `new Throwable().getStackTrace();`

Reply:
good point